front/app/containers/EventsShowPage/components/InformationColumnDesktop.tsx

```diff
@@ -74,6 +75,12 @@ const InformationColumnDesktop = ({ event, className }: Props) => {
                   <Box borderBottom={`solid 1px ${colors.divider}`} />
                 </>
               )}
+              {event.attributes.location_description && (
+                <>
+                  <OnlineLink link={event.attributes.online_link} />
+                  <Box borderBottom={`solid 1px ${colors.divider}`} />
```

Comment:
In this case it's just used as a divider (border provides a small grey line to separate other elements). Though it could be on a box wrapper instead actually - I'll make that change :)